include/__threading_support

```diff
@@ -93,8 +96,14 @@ typedef void* __libcpp_recursive_mutex_t[5];
 #endif
 
 // Condition Variable
+#if defined(NXDK)
+// Size and content of CONDITION_VARIABLE are different in nxdk
+typedef int32_t __libcpp_condvar_t[4];
```

Comment:
Where does this know `int32_t` from? This doesn't seem to include `cstdint`.

Reply:
I changed it to `int` and added a static assert.